insights/combiners/lvm.py

```diff
@@ -242,3 +243,38 @@ def filter_logical_volumes(self, lv_filter, vg_filter=None):
         else:
             return {k: v for k, v in self.logical_volumes.iteritems()
                     if lv_filter in k.LV and vg_filter in k.VG}
+
+
+@combiner(requires=[[LvsAll, PvsAll, VgsAll]])
+class LvmAll(Lvm):
+    """A Lvm like shared combiner for processing LVM information including all rejected
+        and accepted devices"""
+
+    LvVgName = namedtuple('LvVgName', ['LV', 'VG'])
```

Comment:
This `LvVgname` can be removed from the sub-class, and use the Lvm.LvVgname directly.